### DISCUSSION TITLE
Android and focus to InputElements (TextBox) on PointerPressed and scrolling

### DISCUSSION THREAD

**Discussion by msneijders** [AvaloniaUI/Avalonia#12522] [Q&A]
Avalonia on Android (touch screen) seems to give focus to a textbox on PointerPressed immediately; opening the softkeyboard. When I want to start scrolling by swiping, and I touch a textbox at the start, it focuses it.

In my experience, native Android apps give focus on PointerReleased; and only giving some visual feedback with a longpress.

Is this behavior optional in avalonia for Android? Or should we turn this in an issue?

https://github.com/AvaloniaUI/Avalonia/assets/27849065/917527fb-0428-4f0d-a19e-efc8949b21c3

**Comment by timunie** [comment]
@emmauss what do you think, should I convert this into an issue?
